account_id.go

```diff
@@ -126,10 +143,22 @@ func (id AccountID) ToSolidityAddress() string {
 }
 
 func (id AccountID) _ToProtobuf() *services.AccountID {
-	return &services.AccountID{
-		ShardNum:   int64(id.Shard),
-		RealmNum:   int64(id.Realm),
-		AccountNum: int64(id.Account),
+	if id.AliasKey == nil {
```

Comment:
Instead of making two blocks for the entire `AccountID` make an `if` block only for the `AccoutNum` part to de-duplicate at least some of the lines.